tests/unit/test_log.py

```
@@ -67,20 +67,102 @@
+class TestClearOldLogs(unittest.TestCase):
+    def setUp(self):
+        self.logger = log.get_logger(
+            f"s3_upload_{uuid4().hex}", log_level=logging.INFO
+        )
+
+    def tearDown(self):
+        log.clear_old_logs(
+            logger=self.logger, log_dir=TEST_DATA_DIR, backup_count=-1
+        )
+
+    def test_files_older_than_backup_count_are_deleted(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_files_older_than_backup_count_are_deleted`

Add return type annotation: `None`

(ANN201)